extensions/sql-database-projects/src/dialogs/addSqlBindingQuickpick.ts

```diff
@@ -204,7 +219,30 @@ export async function launchAddSqlBindingQuickpick(uri: vscode.Uri | undefined,
 					}
 					if (connectionString) {
 						try {
-							const success = await azureFunctionsUtils.setLocalAppSetting(path.dirname(projectUri.fsPath), newConnectionStringSettingName, connectionString);
+							const projectFolder: string = path.dirname(projectUri.fsPath);
+							const localSettingsPath: string = path.join(projectFolder, constants.azureFunctionLocalSettingsFileName);
+							// Connection string does not include a password, so will prompt user to optionally enter it manually
+							if (!connectionInfo?.password || includePassword === constants.noString) {
+								let userPassword = await vscode.window.showInputBox({
+									prompt: constants.includePassword,
+									placeHolder: constants.enterPasswordManually,
+									ignoreFocusOut: true,
+									password: true
```

Comment:
We should also be validating that the field is truthy. Escape can be used to indicate that they don't want to enter a password right now